lib/radiator_web/graphql/admin/resolvers/editor.ex

```diff
@@ -302,6 +307,10 @@ defmodule RadiatorWeb.GraphQL.Admin.Resolvers.Editor do
     {:ok, Media.NetworkImage.url({network.image, network})}
   end
 
+  def get_image_url(%User{person: person}, _, _) do
+    {:ok, Media.PersonAvatar.url({person.avatar, person})}
+  end
+
   def get_episodes_count(%Podcast{id: podcast_id}, _, %{context: %{authenticated_user: user}}) do
     Editor.get_episodes_count_for_podcast!(user, podcast_id)
   end
```

Comment:
Hmm - was looking for some sort of api to add contributors here. This is still out, is it? I think even if you do it in a subcontext `Radiator.Contribution` you should expose it here, so we can add permission checks to the modification. Also wondering where to put both the `User` and `Person` admin api's in general?

Reply:
Yes, Elixir and gql API are still on the todo, see initial PR description.